GCDWebServer/Core/GCDWebServerRequest.m

```diff
@@ -157,7 +157,9 @@ @interface GCDWebServerRequest () {
   NSString* _noneMatch;
   NSRange _range;
   BOOL _gzipAccepted;
-  
+  NSData* _localAddress;
+  NSData* _remoteAddress;
+
```

Comment:
Sorry I missed this one :sweat_smile: 

What's the problem with it exactly? I mean, the the empty line was there before, so is it just that or is there something I'm missing?